definitions/oi-kamon-agent/definition.yml

```diff
@@ -0,0 +1,18 @@
+domain: OI
```

Comment:
I see the validation failed because the domain is too short. Open to recommendations :)

Reply:
OIT?

Reply:
EXT?